ui/src/svg.d.ts

```diff
@@ -0,0 +1,5 @@
+// svg-shim.d.ts
+declare module "*.svg" {
+  const content: any;
```

Comment:
You can do `// eslint-disable-next-line @typescript-eslint/no-explicit-any` to ignore this `Unexpected any. Specify a different type ` warning.

We lint against this because we typically don't want people to use `any`, we want stronger types than that, but in this case we should make the svg modules as generic as possible.